pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1510,18 +1515,61 @@ func getNetworkProviderConfig(ctx context.Context, configManager ConfigManager,
+	if vpcCreateConfig != nil && vpcCreateConfig.CidrBlock != nil && *vpcCreateConfig.CidrBlock != "" {
+		_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+		if err != nil {
+			return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		}
+		cidrMask, _ := vpcCidr.Mask.Size()
+
+		logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
+		return vpcCidr, nil
+	}
+
+	//if vpcCreateConfig.CidrBlock is nil or an empty string we can go ahead and set a sensible non overlapping default with a mask size of /26
+	defaultCIDR, err := getNonOverlappingDefaultCIDR(ctx, n.Client, n.Ec2Api, n.Logger)
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "failed to generate default CIDR")
 	}
 
-	_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+	// update the config map with the default cidr
+	// if no config map is found an error will be returned as this is required by cro to configure any aws resources
+	err = configManager.ReconcileNetworkStrategy(ctx, n.Client, defaultCIDR.String())
 	if err != nil {
-		return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		return nil, errorUtil.Wrap(err, "failed to reconcile strategy config map")
 	}
-	cidrMask, _ := vpcCidr.Mask.Size()
+	// a default cidr is generated and updated to the config map, that is returned for use in the network creation
+	return defaultCIDR, nil
+}
 
-	logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
-	return vpcCidr, nil
+// getNonOverlappingDefaultCIDR returns a non overlapping cidr block based on the OpenShift vpc cidr block
+// the default mask is /26
+// for other masks the user is required to provide their own via config
+func getNonOverlappingDefaultCIDR(ctx context.Context, client client.Client, ec2Api ec2iface.EC2API, logger *logrus.Entry) (*net.IPNet, error) {
+	clusterVpc, err := getClusterVpc(ctx, client, ec2Api, logger)
+	if err != nil {
+		errMsg := "failed to get cluster vpc for cidr block"
+		return nil, errorUtil.Wrap(err, errMsg)
+	}
+	clusterCIDR := *clusterVpc.CidrBlock
+
+	//parse the cidr to an ipnet cidr in order to manipulate the ip
+	clusterCidrIp, _, err := net.ParseCIDR(clusterCIDR)
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "error parsing cluster cidr block")
+	}
+	ip := clusterCidrIp.To4()
+	// increment the second byte of the ip by 4 bytes or by 1, e.g. if the ip is 10.0.0.0 then we want 10.1.0.0
+	// as the maximum cro allowed mask size is /16, we are using /26 for the default but the following default
+	// will apply in the case of /16 to /26
+	ip[1] = ip[1] + byte(1)
+
+	_, defaultCIDR, err := net.ParseCIDR(fmt.Sprintf("%s/%s", ip, defaultCIDRMask))
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "error parsing standalone default cidr block")
+	}
+	return defaultCIDR, nil
```

Comment:
probably should have some sort of loop here, to validate the `defaultCIDR` is valid and does not overlap with the `clusterCIDR`